ovos_dinkum_listener/voice_loop/voice_loop.py

```
@@ -711,15 +712,15 @@ def _get_tx(self, stt_context: dict) -> (str, dict):
-            utts = self.stt.transcribe() or []
+            utts = self.stt.transcribe(lang=lang) or []
         except:
             LOG.exception("STT failed")
             utts = []
 
         if not utts and self.fallback_stt is not None:
             LOG.info("Attempting fallback STT plugin")
             try:
-                utts = self.fallback_stt.transcribe() or []
+                utts = self.fallback_stt.transcribe(lang=lang) or []
             except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)